app/_includes/nav-v2.html

```diff
@@ -6,16 +6,33 @@
         alt="Kong Logo"
       />
     </a>
+    {% if page.layout != "landing-page" %}
+    <div class="search-input-wrapper">
+      <img src="/assets/images/landing-page/search.svg" alt="search">
+      <input id="getkong-algolia-search-input" type="text" placeholder="Search...">
+    </div>
+    {% endif %}
     <div class="separator"></div>
     <div class="navbar-items">
       <div class="navbar-items">
-        <div class="navbar-item" onclick="toggleSubmenuVisible(this)">
+        <div class="navbar-item{% if page.layout != 'landing-page' %} active{% endif %}" onclick="toggleSubmenuVisible(this)">
           <a href="https://docs.konghq.com/">Docs</a>
           <div class="navbar-item-submenu">
-            <a href="/gateway-oss/" class="navbar-item">Kong Gateway</a>
-            <a href="/konnect/" class="navbar-item">Kong Konnect</a>
-            <a href="/mesh/" class="navbar-item">Kong Mesh</a>
-            <a href="/hub/" class="navbar-item">Plugin Hub</a>
+            <div class="submenu-section">
+              <span class="navbar-item submenu-section-title">Kong Konnect</span>
```

Comment:
The Kong Konnect and Open Source titles are both a bit too small. They're a little smaller than the text below them - not enough to create a lot of contrast, but just enough to feel slightly awkward. Can you increase them one or two points to balance it out? 

Also, can you make these section titles, "On this page" (right sidebar), and first breadcrumb all be same size, all caps, and font weight?